### PR TITLE
put into correct states and fix imports SSCS-10035

### PR DESCRIPTION
**Pull request by nickhill111**
https://tools.hmcts.net/jira/browse/SSCS-10035

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by amoslv1 — APPROVED** [review]

**Review by baskernadarv1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/idam/UserRole.java [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
+@RequiredArgsConstructor
 public enum UserRole {
-    TCW("caseworker-sscs-registrar"),
-    CTSC_CLERK("caseworker-sscs-clerk"),
-    DWP("caseworker-sscs-dwpresponsewriter"),
-    JUDGE("caseworker-sscs-judge"),
-    SUPER_USER("caseworker-sscs-superuser"),
-    SYSTEM_USER("caseworker-sscs-systemupdate"),
-    CITIZEN("citizen");
+    SUPER_USER("caseworker-sscs-superuser", "Super User"),
```
Comment (DGordonV1) [inline src/main/java/uk/gov/hmcts/reform/sscs/idam/UserRole.java:9]: Have these changes been tested fully, always a bit cautious where user role related changes, any knock effects in calls to retrieve the user role anywhere in tribunals?

File: src/main/java/uk/gov/hmcts/reform/sscs/utility/StringUtils.java [hunk 2]
```diff
@@ -1,10 +1,9 @@
 package uk.gov.hmcts.reform.sscs.utility;
 
-import uk.gov.hmcts.reform.sscs.model.client.JudicialUserSearch;
-
 import java.util.Arrays;
 import java.util.Iterator;
 import java.util.List;
+import uk.gov.hmcts.reform.sscs.model.client.JudicialUserSearch;
```
Comment (DGordonV1) [inline src/main/java/uk/gov/hmcts/reform/sscs/utility/StringUtils.java:6]: I'm guessing the changes to import placement are fine and don't break checkstyle stuff?

File: src/main/java/uk/gov/hmcts/reform/sscs/idam/UserRole.java (hunk 1 above)
Comment (nickhill111) [reply src/main/java/uk/gov/hmcts/reform/sscs/idam/UserRole.java:9]: All the values are the same so they should be the same, its just extending them to have labels as well

**Review by DGordonV1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PostHearing.java [hunk 3]
```diff
@@ -88,21 +92,11 @@ public RequestFormat getRequestFormat() {
...
     }
-
-    @JsonIgnore
-    public boolean isRefused() {
-        return SetAsideActions.REFUSE.equals(getSetAside().getAction())
-            || CorrectionActions.REFUSE.equals(getCorrection().getAction())
-            || StatementOfReasonsActions.REFUSE.equals(getStatementOfReasons().getAction())
-            || PermissionToAppealActions.REFUSE.equals(getPermissionToAppeal().getAction())
-            || LibertyToApplyActions.REFUSE.equals(getLibertyToApply().getAction());
-    }
```
[the remark this answers is not in the document]
Comment (nickhill111) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PostHearing.java:107]: Its not used anymore, under the refactoring that we spoke about yesterday, its been removed

**Review by amoslv1 — APPROVED** [review]

**Pull request closed without merging**